gumbo-parser/fuzzer/build.sh

```diff
@@ -1,9 +1,13 @@
 export SANITIZER_OPTS=""
 export SANITIZER_LINK=""
 
+if [ -x "$(command -v llvm-config)" ]; then
```

Comment:
I think this block will overwrite `$LLVM_CONFIG` with `llvm-config` even if I've set the env var. I think it should first check if $LLVM_CONFIG before falling back to `llvm-config`.

Reply:
fixed